playwright/e2e/selection.spec.ts

```diff
@@ -0,0 +1,316 @@
+import { expect, test } from '../support/test-helpers';
+
+test.describe('selection', () => {
+  test.describe('cell selection', () => {
+    const example = 'cell-selection';
+
+    test(example, async ({ si, page }) => {
+      await si.visitExample(example);
+      const nameCell = page.getByRole('cell', { name: 'Ethel Price' });
+      await expect(nameCell).toBeVisible();
+
+      const nameCellParentEl = nameCell.locator('..').locator('..');
+
+      await nameCell.click();
+
+      await expect(nameCell).toHaveClass(/active/);
+      await expect(nameCellParentEl).toHaveClass(/active/);
+
+      await si.runVisualAndA11yTests('name-cell-selection');
```

Comment:
cell selection seems to capture a very large height image due to large number of data. can you ensure to only capture snapshot with predefined height width.